modules/Components/src/client/ACASThingBrowser.coffee

```diff
@@ -69,16 +63,48 @@ class ThingSimpleSearchController extends AbstractFormController
 					values: [
 
 					]
-
+				returnDTO: {
+					thingValues: []
+					thingAttributes: ["id", "codeName"]
```

Comment:
We need codeName and id for the ls thing editor forms when you click on a row. So we always pass these in the Ajax request.